packages/concerto-core/types/lib/introspect/mapkeytype.d.ts

```diff
@@ -17,14 +17,20 @@ declare class MapKeyType extends Decorated {
      */
     constructor(parent: MapDeclaration, ast: any);
     parent: MapDeclaration;
-    type: any;
     /**
      * Semantic validation of the structure of this class.
      *
      * @throws {IllegalModelException}
      * @protected
      */
     protected validate(): void;
+    /**
+     * Sets the Type name for the Map Key
+     *
+     * @param {Object} ast - The AST created by the parser
+     */
+    processType(ast: any): void;
```

Comment:
This should be an internal function only

Reply:
I did intend to use `#private` class feature but it produces the error `Private identifiers are only available when targeting ECMAScript 2015 and higher`.

Is there a convention in place for handling internal functions, or do you have a suggestion? I see `@private` used, but my understanding is this is for doc-gen purposes.